index.js

```diff
@@ -683,4 +683,64 @@ Peer.prototype._debug = function () {
   debug.apply(null, args)
 }
 
+// Transform constraints objects into the new format (unless Chromium)
+// This can be removed when Chromium supports the new format
+function transformConstraints (constraints) {
+  if (typeof (window) === 'undefined' || Object.keys(constraints).length === 0) {
+    return constraints
+  }
+  var isChromium = window.chrome
```

Comment:
isChromium is needed before the constraints are transformed, but the other user agent sniffing is done after the pc is made. I can't them in the same block, but I'll move it up into the constructor.